lib/Rex/Commands/Sysctl.pm

```diff
@@ -95,10 +99,14 @@ sub sysctl {
       Rex::Logger::debug("$key has already value $val");
     }
 
-    if ( $options{persistent} ) {
+    if ( $options{ensure} eq "present" ) {
```

Comment:
Does it work if no options are passed at all (i.e. calling `sysctl` command from existing code)? Wouldn't this complain about `Use of uninitialized value in string eq` or similar due to being undefined?